dbt/models/iasworld/schema/iasworld.htpar.yml

```diff
@@ -303,6 +312,9 @@ sources:
                 - cpaddr3
               allow_interior_space: true
               config: *unique-conditions
+              meta:
+                description: >
+                  Address columns should have no leading or trailing whitespace
```

Comment:
Whitespace failures will be particularly confusing to read in the workbook unless we come up with a way of highlighting them in the output; I wonder if it would be better to just remove these tests entirely?

Reply:
Cool! I'll leave that for a follow-up PR, I think we should probably have a broader discussion with the other teams about what tests to keep or hide since I expect there may be more that they want to get rid of.